crate_ruby.gemspec

```diff
@@ -27,18 +27,34 @@ require 'crate_ruby/version'
 Gem::Specification.new do |spec|
   spec.name          = 'crate_ruby'
   spec.version       = CrateRuby::VERSION
-  spec.authors       = ['Christoph Klocker', 'CRATE Technology GmbH']
+  spec.authors       = ['Christoph Klocker', 'Crate.IO GmbH']
   spec.email         = ['office@crate.io']
-  spec.summary       = 'A simple interface for the Crate.IO database.'
-  spec.description   = 'A Ruby interface for Crate.IO. Put your data to work. Simply.'
-  spec.homepage      = 'http://crate.io'
+  spec.summary       = 'CrateDB HTTP client library for Ruby'
+  spec.description   = 'A Ruby interface for CrateDB featuring query support, DDL command shortcuts, and support for BLOB tables'
+  spec.homepage      = 'https://crate.io'
   spec.license       = 'Apache-2.0'
   spec.required_ruby_version = '>= 2.6'
 
-  spec.files         = `git ls-files -z`.split("\x0")
+  spec.files         = Dir['lib/**/*']
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
-  spec.require_paths = ['lib']
+  spec.require_paths = %w[lib]
+  spec.extra_rdoc_files = Dir['README.rst', 'CHANGES.rst', 'LICENSE', 'NOTICE']
+  spec.rdoc_options    += [
+    '--title', 'CrateDB HTTP client library for Ruby',
+    '--main', 'README.rst',
+    '--line-numbers',
+    '--inline-source',
+    '--quiet'
+  ]
+
+  spec.metadata = {
+    'bug_tracker_uri' => 'https://github.com/crate/crate_ruby/issues',
+    'changelog_uri' => 'https://github.com/crate/crate_ruby/blob/main/CHANGES.rst',
+    'documentation_uri' => 'https://www.rubydoc.info/gems/crate_ruby/0.2.0',
```

Comment:
Can we use `CrateRuby::VERSION` here to prevent the hardcoded version?

Reply:
Thanks for spotting this. Let's just omit the the version number here. Fixed with 3fade6ef.